File: bitget-golang-sdk-api/pkg/model/mix/plan/modifyplanreq.go
package plan

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: placePlan request
 */
type ModifyPlanReq struct {
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
	/**
	 * Deposit currency
	 */
	MarginCoin string `json:"marginCoin"`
	/**
	 * Planned entrusted order No
	 */
	OrderId string `json:"orderId"`
	/**
	 * Execution price
	 */
	ExecutePrice string `json:"executePrice"`
	/**
	 * Trigger Price
	 */
	TriggerPrice string `json:"triggerPrice"`
	/**
	 * Trigger Type
	 */
	TriggerType string `json:"triggerType"`
	/**
	 * Order Type
	 */
	OrderType string `json:"orderType"`
}

File: bitget-node-sdk-api/src/lib/model/mix/plan/ModifyPlanReq.ts
export class ModifyPlanReq{
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * Deposit currency
     */
    private _marginCoin!:string;
    /**
     * Planned entrusted order No
     */
    private _orderId!:string;
    /**
     * Execution price
     */
    private _executePrice!:string;
    /**
     * Trigger Price
     */
    private _triggerPrice!:string;
    /**
     * Trigger Type
     */
    private _triggerType!:string;
    /**
     * Order Type
     */
    private _orderType!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get marginCoin(): string {
        return this._marginCoin;
    }

    set marginCoin(value: string) {
        this._marginCoin = value;
    }

    get orderId(): string {
        return this._orderId;
    }

    set orderId(value: string) {
        this._orderId = value;
    }

    get executePrice(): string {
        return this._executePrice;
    }

    set executePrice(value: string) {
        this._executePrice = value;
    }

    get triggerPrice(): string {
        return this._triggerPrice;
    }

    set triggerPrice(value: string) {
        this._triggerPrice = value;
    }

    get triggerType(): string {
        return this._triggerType;
    }

    set triggerType(value: string) {
        this._triggerType = value;
    }

    get orderType(): string {
        return this._orderType;
    }

    set orderType(value: string) {
        this._orderType = value;
    }
}